src/rpc/blockchain.cpp

```diff
@@ -1320,6 +1330,9 @@ static RPCHelpMan getdeploymentinfo()
                 {RPCResult::Type::OBJ_DYN, "deployments", "", {
                     {RPCResult::Type::OBJ, "xxxx", "name of the deployment", RPCHelpForDeployment}
                 }},
+                {RPCResult::Type::OBJ_DYN, "script_flag_exceptions", "All script flag exceptions for this chain in blocks before or after the tip.", {
```

Comment:
faf3b412e0316f626ceb15658dd9a017cacbf07e:
> `for this network. Exempted blocks are not necessarily part of the chain, e.g. during IBD or an extreme reorg.`

Reply:
I use the word "exempted" here, hopefully correctly: script flag exceptions determine which blocks are exempted from the default script flags.

Reply:
thx, done

Reply:
exempt goes with exemption, except goes with exception - https://www.differencebetween.com/difference-between-exemption-and-vs-exception/

exempt implies fewer restrictions, an exception simply implies different restrictions. either's correct here.

Reply:
I based this on a different explanation: https://english.stackexchange.com/a/419105

But on second read it's not that clear. Picking one sounds good to me too.